www/cherokee/Portfile

```diff
@@ -32,49 +71,39 @@ openssl.branch 1.0
 openssl.configure build_flags
 
 # Startup item.
-set cherokee_config_name   cherokee.conf
-set cherokee_config        ${prefix}/etc/${name}/${cherokee_config_name}
-set cherokee_pidfile_name  cherokee.pid
-set cherokee_pidfile       ${prefix}/var/run/${cherokee_pidfile_name}
-
-startupitem.create  yes
-startupitem.init    "PIDFILE=${cherokee_pidfile}"
-startupitem.start   "${prefix}/sbin/cherokee -C ${cherokee_config}"
-startupitem.stop    "kill \$(cat \$PIDFILE)"
-
-variant no_startupitem description {Do not create a startup item} {
-    startupitem.create  no
+set cherokee_config_name    cherokee.conf
+set cherokee_config         ${prefix}/etc/${name}/${cherokee_config_name}
+set cherokee_pidfile_name   cherokee.pid
+set cherokee_pidfile        ${prefix}/var/run/${cherokee_pidfile_name}
+
+startupitem.create          yes
+startupitem.init            "PIDFILE=${cherokee_pidfile}"
+startupitem.start           "${prefix}/sbin/cherokee -C ${cherokee_config}"
+startupitem.stop            "kill \$(cat \$PIDFILE)"
```

Comment:
There shouldn't need to be quotation marks around any of these.

Reply:
Fixed.